### PR TITLE
Add 28-Day Active Telemetry

### PR DESCRIPTION
**Pull request by utsavized**
Add 28-day anonymous active user count for online scanner. Routes activity data through internal API to drop extra metrics that App Insights automatically sends.

### CODE REVIEW

File: src/webhint-theme/source/js/telemetry/telemetry.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
+"use strict";
+(function () {
+    const activityKey = 'webhint-activity';
+    const productKey = 'webhint-online-scanner';
+    const storage = window.localStorage;
+    const telemetryApiEndpoint = 'https://webhint-telemetry-api-appservice.azurewebsites.net/api/webhint-telemetry-ingress-api';
```
Comment (antross) [inline src/webhint-theme/source/js/telemetry/telemetry.js:6]: Could this submit to an endpoint served as part of our existing telemetry repo instead? https://github.com/webhintio/telemetry

That already contains Azure Functions which work with our Application Insights instance. I was hoping we could add a new function there for this endpoint that we can share between here and our extensions.

Comment (utsavized) [reply src/webhint-theme/source/js/telemetry/telemetry.js:6]: https://github.com/webhintio/telemetry/pull/164

**Comment by utsavized** [comment]
@antross endpoint updated

File: src/webhint-theme/source/js/telemetry/telemetry.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
+"use strict";
+(function () {
```
Comment (antross) [inline src/webhint-theme/source/js/telemetry/telemetry.js:2]: Should we log a task/bug to track sourcing this from the same code as @hint/utils-telemetry (used by the VS Code and browser extensions) once that's been updated to use the new endpoint? Probably also need a task to track migrating that code in the first place.

File: src/webhint-theme/source/js/telemetry/telemetry.js [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+     * Report once per UTC day that a user is active (has run a scan).
+     * Data includes `last28Days` (e.g. `"1001100110011001100110011001"`)
+     * and `lastUpdated` (e.g. `"2019-10-04T00:00:00.000Z"`).
+     */
+    // Don't count a user as active if telemetry is disabled.
+
+    const activity = getUpdatedActivity(storage.getItem(activityKey));
+    if (activity) {
+        storage.setItem(activityKey, JSON.stringify(activity));
+        trackEvent('online-scanner-activity', activity);
```
Comment (antross) [inline src/webhint-theme/source/js/telemetry/telemetry.js:112]: The prefix we've been using for telemetry related to the online scanner is simply "online-".
```suggestion
        trackEvent('online-activity', activity);
```

E.g.:
https://github.com/webhintio/serverless-online-service/blob/62528ec3e1eea06e1501b647e2f9a1c07e7d93c1/packages/services/src/sync/sync-service.ts#L250

File: src/webhint-theme/source/js/telemetry/telemetry.js [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+    }
+}());
```
Comment (antross) [inline src/webhint-theme/source/js/telemetry/telemetry.js:114]: Nit: keep a newline at the end of the file for consistency.

**Pull request merged**